docs/archive-data.md

```diff
@@ -0,0 +1,314 @@
+| `namespaceDefaults.archival.history.state`     | Acceptable values are `"enabled"`, `"disabled"`. This sets the default state of the Archival feature whenever a new Namespace is created without specifying the Archival state. |
+| `namespaceDefaults.archival.history.URI`       | The value must be a URI of the file store location and match a schema that correlates to a provider. |
+
+Consider the following example:
+
+```yaml
+# Cluster level Archival config
+archival:
+  # Event History configuration
+  history:
+    # State at the cluster level
+    state: "enabled"
+    # Read permissions
+    enableRead: true
+    # Available providers
+    provider:
+      # Local filestore provider  
+      filestore:
+        # File permissions
+        fileMode: "0666"
+        # Directory permissions
+        dirMode: "0766"
+      # Google Cloud provider  
+      gstorage:
+        # Google Cloud credentials file
+        credentialsPath: "/tmp/gcloud/keyfile.json"
+
+# Default values for a Namespace if none are provided at creation
+namespaceDefaults:
+  # Archival defaults
+  archival:
+    # Event History defaults
+    history:
+      # Default state
+      state: "enabled"
+      # Default URI
+      URI: "file:///tmp/temporal_archival/development"
+```
+
+In the configuration above we see that:
+
+1. Archival is enabled at the cluster level.
+2. Namespaces can use either the local filestore provider or the Google Cloud provider.
+3. New Namespaces will default to the local provider, 
+
+:::tip
+
+**Want to disable Archival?**
+
+You can disable Archival by setting `archival.history.state` and `namespaceDefaults.archival.history.state` to `"disabled"`.
+
+Example:
+
+```yaml
+archival:
+  history:
+    state: "disabled"
+
+namespaceDefaults:
+  archival:
+    history:
+      state: "disabled"
+```
+
+:::
+
+### Namespace creation
+
+The Archival URI is set when a Namespace is created. If you do not specify the URI when creating a Namespace, it will use the value of `defaultNamespace.archival.history.URI` from the `config/development.yaml` file. Once set, the URI can not be changed. Each Namespace supports a single Archival URI, but each Namespace can use a different URI (archiver).
+
+At the cluster level, `archival.history.state` must be set to `"enabled"` for Archival to work at the Namespace level for Even Histories. Once enabled at the cluster level, a Namespace can safely switch between `enabled` and `disabled` states.
+
+Archival is supported in global Namespaces (Namespaces spanning multiple clusters). When Archival is running in a global Namepsace, it will first run on the active cluster and then some time later it will run on the standby cluster. Before archiving, a history check is done to see what has been previously uploaded.
+
+## Archive reads
+ 
+To retrieve archived Event Histories you will need the Workflow Id and the Run Id:
+
+```bash
+$ ./temporal --ns <namespace> wf show --wid <workflowId> --rid <runId>
```

Comment:
unrelated, can code snippets have copy buttons? (and the copy shouldn't take the $).